.github/actions/semantic-pr-footer-v1/package.json

```
@@ -18,5 +18,24 @@
     "coverage": "nyc npm run test",
     "fmt": "prettier --write \"src/**/*.{js,ts,tsx,json,md,css,html}\"",
     "lint": "eslint \"src/**/*.{js,ts,tsx}\""
+  },
+  "nyc": {
+    "checkCoverage": true,
+    "extension": [
+      ".ts"
+    ],
+    "reporter": [
+      "text-summary",
+      "html"
+    ],
+    "statements": 90,
+    "branches": 90,
+    "functions": 90,
+    "lines": 90,
```

nit - feel free to ignore:

```suggestion
    "statements": 100,
    "branches": 100,
    "functions": 100,
    "lines": 100,
```